VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -571,6 +571,15 @@
   <data name="UsingStatementWithoutBracesCodeFixTitle" xml:space="preserve">
     <value>Add braces</value>
   </data>
+  <data name="ImplementEqualsAndGetHashCodeAnalyzerMessage" xml:space="preserve">
+    <value>Class {0} does not implement Equals() and GetHashCode().</value>
```

Comment:
Should just change it to `Member`, or should I explicitly say "Class" or "Struct" as applicable?